cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ApplicationsOperations.java

```diff
@@ -67,26 +64,32 @@ public ApplicationsOperations(DefaultCloudFoundryOperations cloudFoundryOperatio
                         .builder()
                         .name(applicationSummary.getName())
                         .build());
+
     }
 
     /**
      * Deletes a specific application associated with the name <code>applicationName</code>.
```

Comment:
We might want to mention, that the mono will take care of logging if the removal was successful.